### PR TITLE
fixed: initialize members

### PR DESCRIPTION
**Pull request by akva2**
caused issues for serialization tests in the debug iterator build

### CODE REVIEW

File: opm/models/blackoil/blackoilsolventparams.hpp [hunk 1]
```diff
@@ -59,30 +59,30 @@ struct BlackOilSolventParams
...
+    std::vector<Scalar> tlMixParamViscosity_{}; // Todd-Longstaff mixing parameter for viscosity
+    std::vector<Scalar> tlMixParamDensity_{};   //  Todd-Longstaff mixing parameter for density
+    std::vector<TabulatedFunction> tlPMixTable_{}; // the tlpmixpa(Po) column of the TLPMIXPA table
+
+    bool isMiscible_{};
     bool rsSolw_active_ = false;
-    bool co2sol_;
-    bool h2sol_;
+    bool co2sol_{};
+    bool h2sol_{};
```
Comment (bska) [inline opm/models/blackoil/blackoilsolventparams.hpp:85]: Bools are kind of tricky and we should probably ensure that value intialisation does what we want in this instance.  On the other hand, it's probably no worse than what we already have so I guess it's not a big deal.

**Review by bska — APPROVED** [review]
With the possible exception of using value initialisation for `bool`s, this looks good to me and we should merge it.

File: opm/models/blackoil/blackoilsolventparams.hpp (hunk 1 above)
Comment (akva2) [reply opm/models/blackoil/blackoilsolventparams.hpp]: sure i can write the = false ;)

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  Please merge when green.

**Pull request merged**